third_party/move/move-compiler/transactional-tests/tests/control_flow/break_continue_for_loop_nested.move

```diff
@@ -0,0 +1,28 @@
+//# run
+script {
+    fun main() {
+        let k = 0;
+	let y = 0;
+	for (j in 0..10) { //j:  0,  1,  2,  3,  4, 5, 6, 7, 8, 9, 10
```

Comment:
The indentation in this test is broken: makes it hard to read.

Reply:
again, tab issue.